resources/views/admin/blocks/showcase.blade.php

```diff
@@ -1,74 +1,228 @@
+@php
+    $currentUrl = explode('/', request()->url());
+    $type = \App\Models\LandingPage::find(intval($currentUrl[4]))->getType();
+
+    switch ($type) {
+        case 'Home':
+            $themes = ['default', 'home'];
+            $mediaTypes = ['image'];
+            break;
+        case 'Visit':
+            $themes = ['default', 'visit'];
+            $mediaTypes = ['image'];
+            break;
+        case 'RLC':
+            $themes = ['default', 'rlc', 'multiple'];
```

Comment:
"multiple" isn't really a theme, it's a distinct block, different from the regular showcase block.